cmd/chunks-inspect/main.go

```diff
@@ -67,65 +65,64 @@ func printFile(filename string, blockDetails, printLines, storeBlocks bool) {
 
 	fmt.Println("Format (Version):", lokiChunk.format)
 	fmt.Println("Encoding:", lokiChunk.encoding)
-	fmt.Print("Blocks Metadata Checksum: ", fmt.Sprintf("%08x", lokiChunk.metadataChecksum))
-	if lokiChunk.metadataChecksum == lokiChunk.computedMetadataChecksum {
-		fmt.Println(" OK")
-	} else {
-		fmt.Println(" BAD, computed checksum:", fmt.Sprintf("%08x", lokiChunk.computedMetadataChecksum))
-	}
 	if blockDetails {
 		fmt.Println("Found", len(lokiChunk.blocks), "block(s)")
 	} else {
 		fmt.Println("Found", len(lokiChunk.blocks), "block(s), use -b to show block details")
 	}
-	if len(lokiChunk.blocks) > 0 {
-		fmt.Println("Minimum time (from first block):", time.Unix(0, lokiChunk.blocks[0].minT).In(timezone).Format(format))
-		fmt.Println("Maximum time (from last block):", time.Unix(0, lokiChunk.blocks[len(lokiChunk.blocks)-1].maxT).In(timezone).Format(format))
-	}
```

Comment:
I guess it's safe to remove all these checks because `parseLokiChunk` now uses the `chunkenc` package which contains all the same error checking?

Reply:
That's right. All checks are done by the implementation from chunkenc.